cadc-vos/src/main/java/ca/nrc/cadc/vos/Transfer.java

```diff
@@ -254,60 +212,49 @@ public void setRemoteIP(String remoteIP) {
      * in the given order.
      * @return
      */
-    public List<String> getAllEndpoints()
-    {
+    public List<String> getAllEndpoints() {
         List<String> rtn = new ArrayList<String>();
-        if (this.protocols != null)
-        {
-            for (Protocol p : this.protocols)
-            {
+        if (this.protocols != null) {
```

Comment:
Can remove this null check (and the one on targets)

Reply:
Except protocols can be set to null in the calling code.?

Reply:
done